docs/reference/sql/functions/grouping.asciidoc

```diff
@@ -76,3 +76,8 @@ Instead one can rewrite the query to move the expression on the histogram _insid
 ----
 include-tagged::{sql-specs}/docs.csv-spec[histogramDateTimeExpression]
 ----
+
+[NOTE]
+When the histogram in SQL is applied on **DATE** type instead of **DATETIME**, then the interval specified is rounded to
```

Comment:
I would make this `IMPORTANT`.
Also drop the "then" : `when... **DATETIME**, the interval".